certbot/storage.py

```diff
@@ -281,6 +282,15 @@ def archive_dir(self):
             return os.path.join(
                 self.cli_config.default_archive_dir, self.lineagename)
 
+    @property
+    def is_test_cert(self):
+        """Returns true if this is a test cert from a staging server."""
+        server = self.configuration["renewalparams"].get("server", None)
+        if server:
```

Comment:
That's the common case. If it's left as the default value, it won't be saved to the renewal conf file, so it won't be in the `renewalparams` object that's read back, so the `.get()` method above will set it to none.